docs/en_US/Compression/QuickStart.rst

```diff
@@ -31,7 +31,7 @@ The specification of configuration can be found `here <./Tutorial.rst#specify-th
 Step2. Choose a pruner and compress the model
```

Comment:
I suggest that we can put the notebook into documentation to increase visibility.

Reply:
good suggestion, add this into the doc.